File: curso phyton cisco/calculadora.py
def  main():
    print("PROGRAMA QUE SIMULA UNA CACULADORA \n")
    while(True):
        usuario = int(input("\n Por favor ingrese 1 para comenzar/realizar otra operacion o 2 para salir: "))
        
        while(usuario > 2 or usuario < 1):
            usuario = int(input("\n Por favor ingrese 1 para comenzar/realizar otra operacion o 2 para salir, no numeros mayores que 2 ni menores que 1: "))
       
        suma = int(1)
        resta = int(2)
        multi = int(3)
        div = int(4)

        if usuario == 1:
            print("""\n Operaciones que realiza la calculadora
                        1- es para suma
                        2- es para resta
                        3- es para multiplicar
                        4- es para dividir
            """)

            usuario = int(input("Por favor selecciona que operacion deseas realizar: "))

            while(usuario < 1 or usuario > 4):
                print("""\n Operaciones que realiza la calculadora
                        1- es para suma
                        2- es para resta
                        3- es para multiplicar
                        4- es para dividir
                """)
                usuario = int(input("\n Por favor ingrese un identificador valido de la operacion que desea realizar, deacuerdo al menu ya presentado anteriormente: "))
            
            if usuario == 1:
                num1 = int(input("\nIngrese el primer numero: "))
                num2 = int(input("Ingrese el segundo numero: "))
                print("\n El resultado de la suma es: ",num1 + num2)
            if usuario == 2:
                num1 = int(input("\nIngrese el primer numero: "))
                num2 = int(input("Ingrese el segundo numero: "))
                print("\n El resultado de la resta es: ",num1 - num2)
            if usuario == 3:
                num1 = int(input("\nIngrese el primer numero: "))
                num2 = int(input("Ingrese el segundo numero: "))
                print("\n El resultado de la multiplicacion es: ",num1 * num2)
            if usuario == 4:
                num1 = float(input("\nIngrese el primer numero: "))
                num2 = float(input("Ingrese el segundo numero: "))
                print("\n El resultado de la division es: ",num1 / num2)
        else:
            break

if __name__ == '__main__': 
    main()

File: ejercicios C++/calculadora.cpp
#include <iostream>

using namespace std;

int main()
{
	float  emp, usuario, num1, num2, suma, resta, multi, div;
	
	suma = 1;
	resta = 2;
	multi = 3;
	div = 4;
	emp = 5;
	
	cout << "***Programa que realiza operaciones aritmeticas basicas***" << endl;
	
	while(true)
	{
		cout  << endl << endl << "Presione 5 para empezar o 6 para salir:  ";
		cin >> usuario;
		if(usuario == 5)
		{
			cout << endl << suma << "-para sumar" << endl;
			cout << resta << "-para restar" << endl;
			cout << multi << "-para multiplicar" << endl;
			cout << div << "-para dividir" << endl;
			cout << endl << "Eliga que operacion quiere realizar: "; 
			cin >> usuario;;
	
			cout << endl << "Ingresa primer numero: ";
			cin >> num1;
	
			cout << endl <<  "Ingresa segundo numero: ";
			cin >> num2;
	
			if(usuario == 1)
			{
				cout << endl << "el resultado es " << (num1 + num2) << endl;
				cout << endl << "Proceso seleccionado: suma";
			}
			if(usuario == 2)
			{
				cout << endl << "el resultado es " << (num1 - num2) << endl;
				cout << endl << "Proceso seleccionado: resta";
			}
			if(usuario == 3)
			{
				cout << endl << "el resultado es " << (num1 * num2) << endl;
				cout << endl << "Proceso seleccionado: multiplicacion";
			}
			if(usuario == 4)
			{
				cout << endl << "el resultado es " << (num1 / num2) << endl;
				cout << endl << "Proceso seleccionado: division";
			}
		}
		if(usuario == 6)
		{
			break;
		}
	}
	
	return 0;
}
